docs/faq/glossary.md

```diff
@@ -0,0 +1,630 @@
+such as flights, hotels, tours, or other activities. This has a specific
+implementation that is developed and managed by C4T.
+
+## Builder
+
+Partner building dApps or an integration with the Camino Network.
+
+## Camino Messenger
+
+An efficient travel data exchange system built on Camino Network for instant
+communication between suppliers and distributors.
+
+## Camino Messenger Bot
+
+A client-implementation for sending and receiving messages via the Camino Messenger.
+This application acts as an abstraction layer for ease of communication even if the
+underlying technology stack changes and evolves over time. It offers endpoints to
+communicate with the Camino Messenger network and can actively send requests to
+so-called [partner plugins](#partner-plugin).
+
+## Camino Messenger server
+
+The backbone of the Camino Messenger infrastructure. Essentially, it's a server
+relaying messages between bots. Currently, Matrix messenger server is used.
+
+## Camino Messenger Protocol
+
+Camino Messenger Protocol (CMP) was developed as the communication protocol for
+Camino Messenger. It is used for (de)serializing messages between
+[bots](#camino-messenger-bot) and [partner plugins](#partner-plugin)
+
+## Camino Network
+
+Comprehensive ecosystem with a vast amount of industry partners designed
+specifically for the travel industry leveraging blockchain technology
+
+## Camino Partner
+
+A person or a company that is building on the Camino blockchain which has become a
+Validator and/or a Supporter
+
+## Centralization
+
+The process or result of concentrating decision-making power, authority, or
+functions within a single individual, group, or location. This organizational
+structure is commonly observed in governments, large corporations, and financial
+institutions, where control is exercised from a central point, often leading to a
+hierarchical system of management and administration.
+
+## Centralized System
+
+A centralized system is one in which a single entity or a small group of entities
+have control over the entire system. All data, processes, and controls are managed
+from a central point.
+
+**Key Characteristics:**
+
+1. **Single Point of Control:** One central authority manages the system.
+2. **Data Centralization:** All data is stored in a central database.
+3. **Ease of Management:** Centralized systems are easier to manage because there is
+   a single point of control.
+4. **Efficiency:** Centralized systems can be more efficient in decision-making and
+   resource allocation since decisions are made by a central authority.
+5. **Security Risks:** If the central point is compromised, the entire system can be
+   affected.
+6. **Scalability Issues:** Centralized systems can face challenges in scaling up as
+   the central node can become a bottleneck.
+7. **Vulnerability to Failures:** Centralized systems are vulnerable to single
+   points of failure, meaning if the central node fails, the entire system can be
+   disrupted.
```

Comment:
are point6  and 7 true? 

Is a centralized system not as easily scalable as a decentralized system? With clouds, maybe even easier?
A failure is often caused by a code change with unforeseen side effects. A decentralized system is equally vulnerable to that, right?